apple-pie/styles/style.css

```diff
@@ -0,0 +1,58 @@
+@import url("https://fonts.googleapis.com/css?family=Roboto+Condensed:700|Roboto:100,300,700");
+
+body {
+  font-family: Roboto;
+}
+
+h1 {
+  text-align: center;
+  position: relative;
+  top: 400px;
+  color: white;
+  font-size: 50px;
+  letter-spacing: 0.9px;
+}
+header {
+  width: 960px;
+  height: 900px;
+  background: center / contain url("../images/apple-pie.jpg") no-repeat;
+  margin: auto;
+}
+
+section {
+  width: 600px;
+  border-bottom: 1px solid black;
+  /*margin-left: 200px;*/
+  margin: 10px auto;
+}
```

Comment:
You could also add some line-height here to add more space between text lines
`line-height: 25px;`